articles/ai-services/openai/includes/spring.md

```diff
@@ -73,112 +73,110 @@ ai-completion-demo/
 
    ```xml
    <?xml version="1.0" encoding="UTF-8"?>
-   <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-       xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
-       <modelVersion>4.0.0</modelVersion>
-       <parent>
-           <groupId>org.springframework.boot</groupId>
-           <artifactId>spring-boot-starter-parent</artifactId>
-           <version>3.2.0</version>
-           <relativePath/> <!-- lookup parent from repository -->
-       </parent>
-       <groupId>com.example</groupId>
-       <artifactId>ai-completion-demo</artifactId>
-       <version>0.0.1-SNAPSHOT</version>
-       <name>AICompletion</name>
-       <description>Demo project for Spring Boot</description>
-       <properties>
-           <java.version>17</java.version>
-       </properties>
-       <dependencies>
-           <dependency>
-               <groupId>org.springframework.boot</groupId>
-               <artifactId>spring-boot-starter</artifactId>
-           </dependency>
-           <dependency>
-               <groupId>org.springframework.experimental.ai</groupId>
-               <artifactId>spring-ai-azure-openai-spring-boot-starter</artifactId>
-               <version>0.7.0-SNAPSHOT</version>
-           </dependency>
-           <dependency>
-               <groupId>org.springframework.boot</groupId>
-               <artifactId>spring-boot-starter-test</artifactId>
-               <scope>test</scope>
-           </dependency>
-       </dependencies>
-       <build>
-           <plugins>
-               <plugin>
-                   <groupId>org.springframework.boot</groupId>
-                   <artifactId>spring-boot-maven-plugin</artifactId>
-               </plugin>
-           </plugins>
-       </build>
-       <repositories>
-           <repository>
-               <id>spring-snapshots</id>
-               <name>Spring Snapshots</name>
-               <url>https://repo.spring.io/snapshot</url>
-               <releases>
-                   <enabled>false</enabled>
-               </releases>
-           </repository>
-       </repositories>
-   </project>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```

Comment:
This whole snippet should be indented 3 spaces to align with the triple-back-tick delimiters. 

Also, the indentations within the XML should be 4 spaces.